eiger_io/fs_handler.py

```diff
@@ -186,3 +187,12 @@ def get_file_list(self, datum_kwargs):
             filenames.append(filename)
 
         return filenames
+    
+    def get_file_size(self):
```

Comment:
let's keep the API similar to `get_file_list` which takes a list of datum kwargs for each datum.
define `get_file_size(self, datum_kwargs):`